.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.20.x, 1.21.x]
+        go-version: [1.20.x, 1.21.x, 1.22.x]
```

Comment:
Can we leave a comment reminding maintainers to also keep `release.yaml` updated when changing Go versions?

Reply:
Great idea